src/components/notifications/SMSPartner.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div class="mb-3">
+        <label for="smspartner-key" class="form-label">{{ $t("smspartnerApikey") }}</label>
+        <HiddenInput id="smspartner-key" v-model="$parent.notification.smspartnerApikey" :required="true" autocomplete="new-password"></HiddenInput>
+        <div class="form-text">
+            {{ $t("smspartnerApiurl") }}
+            <a href="https://my.smspartner.fr/dashboard/api" target="_blank">https://my.smspartner.fr/dashboard/api</a>
+        </div>
+    </div>
+    <div class="mb-3">
+        <label for="smspartner-phone-number" class="form-label">{{ $t("smspartnerPhoneNumber") }}</label>
+        <input id="smspartner-phone-number" v-model="$parent.notification.smspartnerPhoneNumber" type="text" class="form-control" required>
```

Comment:
Can/should we restrict what can be entered into this input further?
What is the allowed format of phone numbers according to their docs?

Reply:
In the doc, numbers can be :
- in national (06xxxxxxxx) or international (+336xxxxxxxx) format, for French numbers.
- international format (+496xxxxxxxx), for numbers outside France.

Maybe we can use a pattern like this one :
```
minlength="3" maxlength="20" pattern="^[\d+,]+$"
```

Reply:
Sounds like a good regex

Reply:
Do you think adding a help text that national French numbers and international numbers are allowed is beneficial?

Reply:
I think the "smallest common denominator" is the number in their international format. Maybe just indicate that this is the form in which it should be supplied (international format). I commit in this way. Thank you :)

Reply:
Done in [389353b](https://github.com/louislam/uptime-kuma/pull/4769/commits/389353b4639151012d4d7afb8de6dbec0bc9240c) :)